colorDC1/Dockerfile

```diff
@@ -128,13 +126,18 @@ RUN echo "gea_user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 # Switch to GEA user
 COPY ./jupyter_notebook_config.json /opt/conda/etc/jupyter/jupyter_notebook_config.json
 COPY ./jupyter_notebook_config.py /home/gea_user/.jupyter/jupyter_notebook_config.py
-COPY ./.bashrc /home/gea_user/.bashrc
-RUN chown -R gea_user:gea_user /home/gea_user/.jupyter
-RUN chmod -R 777 /home/jovyan
-RUN chown gea_user:gea_user /home/gea_user/.bashrc
+COPY ./_bashrc /home/gea_user/.bashrc
+COPY ./_bash_prompt /home/gea_user/.bash_prompt
```

Comment:
`_bashrc` in this repository was pretty close to the one in `/etc/skel/.bashrc`, so I made a few modifications to it and copied it in. `_bash_prompt` contains the prompt customization.